src/vte.c

```diff
@@ -221,6 +224,78 @@ WRAP_RGBA_SETTER(vte_terminal_set_color_foreground)
 #endif
 
 
+#if GTK_CHECK_VERSION(3, 0, 0)
+static void default_vte_terminal_set_background_image_file(VteTerminal *terminal, const char *path)
+{
+	if (background_image_pixbuf)
+	{
+		g_object_unref(background_image_pixbuf);
+		background_image_pixbuf = NULL;
+	}
+
+	if (path != NULL)
+	{
+		GError *error = NULL;
+
+		background_image_pixbuf = gdk_pixbuf_new_from_file(path, &error);
+		if (! background_image_pixbuf)
+		{
+			g_warning("Failed to load VTE background image: %s", error->message);
```

Comment:
BTW, apparently VTE's implementation of this fails silently when the file can't be open/isn't an image.  Warning is better for us I suppose, though.